config/manifests/bases/lightspeed-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: openshift-lightspeed
```

Comment:
Does it make sense to add `repository: https://github.com/openshift/lightspeed-operator` annotation?

Reply:
Will this field appear in the console UI?

Reply:
Yes
![Screenshot from 2024-03-04 16-34-06](https://github.com/openshift/lightspeed-operator/assets/89217163/0f17a291-e39c-4c18-93b1-271fc7375e20)